workflow/rtma3d_pbspro_conus.xml

```diff
@@ -904,7 +922,7 @@
 
   <cycledef group="rap">0 0,12 * 01-12 2022-2025 *</cycledef>
 
-  <task name="&RUN;_obsprep_lghtn_task_@Y@m@d@H@M" cycledefs="hourly" maxtries="&maxtries;">
```

Comment:
Hi Gang. I think that the date/time string should be kept with each task name since multiple cycles can and do run at the same time. It would be difficult to distinguish which cycle is running for a given workflow step and if there is a problem, it may be hard to distinguish in which cycle it occurred.

Reply:
@EdwardColon-NOAA I agree with removing the date/time string from the task name.  Whenever I rocotoboot, I have to do the following:

rocotoboot -w rtma3d_pbspro_conus.xml -d rtma3d_pbspro_conus.db -c 202503051100 -t rtma3d_prdgen_task_@Y@m@d@H@M 

as "@Y@m@d@H@M" is part of the task name (and it does not fill in the date).  If you look in Gang's log directories, it does have the date in the logfile name.  So I think it's unneccesary to include "@Y@m@d@H@M" in the task name.

Reply:
@EdwardColon-NOAA ,
Hi Ed,

As @AnnetteGibbs-NOAA mentioned in her comments above, the time stamp string "_@Y@m@d@H@M" actually does NOT work. It in fact did not show the time  string in the task name (because the "<cyclestr> is NOT added in the task name definition yet). But it is not necessary, since the time stamp already show for each task when rocoto lists each task. 

And removing this time stamp in task name has no influence to the log file name. The log file still has the time stamp (since the jobname is defined with timestamp as follows):

`<jobname><cyclestr>&RUN;_obsprep_cloud_job_@Y@m@d@H@M</cyclestr></jobname>`

So I believe it is good to remove the time stamp in the task name definition. 

Thanks,
-Gang